### PR TITLE
Space Dampening and Salvager Docking

### PR DESCRIPTION
**Pull request by Kesiath**
## About the PR
This PR changes a major game mechanic, and also introduces two new flatpack items: the iron mass and docking magnet. The iron mass and docking magnet both act as docks, but can be wrenched, rotated, and moved into position both on and off your vessel or salvage/asteroids.

This PR also disabled linear speed dampening so your flight in space does not slow down like it currently does. This applies to all unanchored grids, so ships and asteroids/salvage. Rotational dampening was increased quite a bit so ships will stop spinning much faster, as will asteroids/salvage.

Huge thanks to Stagnation in the discord for help with the sprites.

## Why / Balance
Space should be dangerous, and the fear of losing your vessel while on EVA should be ever-present. Taking the time to do salvage/mining right with safety equipment (the new docking tools, and/or a grappling hook) and keeping a pilot onboard to make sure your ship doesn't go drift off into space should be more important in our RP and game mechanics.

## How to test
Spawn a ship to test the space changes, and use the flatpack vendor to get the two new machines. The new machines work the same as docks.

## Media
image

**Changelog**
:cl:
- add: Added the iron mass and docking magnet, new salvager tools.
- tweak: Changed the space drag in the sector, so now you hold all linear momentum.

### CODE REVIEW

**Comment by UncaughtEx** [comment]
It serves well to be prepared!

**Comment by Green-00ff00** [comment]
My largest reservation is if a pilot/captain becomes incapacitated at the helm; they'd effectively continue on indefinitely until some external force stops them. That could be something to make medical/SAR a lot more annoying and fiddly, which may not be considered a good thing.

File: Content.Server/Shuttles/Components/ShuttleComponent.cs [hunk 1]
```diff
@@ -66,9 +66,9 @@ public sealed partial class ShuttleComponent : Component
         /// Damping applied to the shuttle's physics component when not in FTL.
         /// </summary>
         [DataField("linearDamping"), ViewVariables(VVAccess.ReadWrite)]
-        public float LinearDamping = 0.05f;
+        public float LinearDamping = 0.00f;
```
Comment (dvir001) [inline Content.Server/Shuttles/Components/ShuttleComponent.cs:69]: Im not sure if its a good idea to have this 0 as possible lag issue?
Also in general this should be a per ship toggle options

Comment (Kesiath) [reply Content.Server/Shuttles/Components/ShuttleComponent.cs:69]: yeah in testing things infinitely floating off does have some issues, that said the current value of .05 feels like way too much, .002 has a lot of drift while definitely slowing vehicles down over a while so I'll probably switch it over to that for the PR

**Comment by Kesiath** [comment]
apparently impossible to just make asteroids innate heavier so their dampening is just a bit higher then ships, although they still float for quite a ways - ships float more naturally and of course retain their speed better - ultimately, and maybe done with this PR if I can figure it out, would be implementing a UI button on the ship, or a machine, which controls dampening (0 for cruising, 0.05 for maneuvering, 0.95 for parking perhaps?)

**Comment by dvir001** [comment]
Another PR aim to patch this issue

**Pull request closed without merging**